public/app/features/integrated-alerting/components/AlertRules/__mocks__/alertRulesStubs.ts

```diff
@@ -35,6 +36,8 @@ export const rulesStubs: AlertRulesListResponseRule[] = [
       },
     ],
     template: {
+      name: 'test_template',
```

Comment:
`AlertRules.utils.test.ts ` needs to be changed as well to expect this new values otherwise the tests fail.

Reply:
Done